src/popup.js

```diff
@@ -102,7 +105,7 @@ const getCookieText = async (format, details) => {
 /** Set URL in the header */
 getUrl.then(url => {
   const location = document.querySelector('#location');
-  location.textContent = location.href = new URL(url).href;
+  location.textContent = new URL(url).hostname;
```

Comment:
```suggestion
  location.textContent = location.href = new URL(url).href;
```
Sorry,
The reason the full URL is in the header text (unlike "Get Cookies.txt") is that the "path" option may make different cookies list, between URLs that are same origin but different path.
Also, in many cases the full URL is over width, so it keeps in the href.